fuzzers/int_maketodo.py

```diff
@@ -209,12 +209,20 @@ def run(
         if side == "r" and not r:
             continue
 
-        if side != "":
-            side = "_" + side
+        if side == "xl":
+            filename = "l{}".format(pip_type)
+        elif side == "xr":
+            filename = "r{}".format(pip_type)
+
+        if side == "l" or side == "r":
+            filename = "{}_{}".format(pip_type, side)
+
+        if side == "":
```

Comment:
Can we make this an if/elif/else tree, with the else an assert so it fails at the end of the if/else tree?  The previous error was "filename var does not exist" type error.

Reply:
Done